presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParser.java

```diff
@@ -2693,7 +2693,7 @@ private static void assertStatement(String query, Statement expected)
         assertFormattedSql(SQL_PARSER, expected);
     }
 
-    private static void assertInvalidStatemennt(String statement, String expectedErrorMessageRegex)
```

Comment:
This is a good typo to fix, but can you do it in a separate commit, since it's unrelated to the materialized view change? It's fine to leave it in this PR as a separate commit.

Reply:
ok, done.